### PR TITLE
Add a note about trusted organizations

### PR DESCRIPTION
**Pull request by cchiffersMSFT**

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/meeting-policies-participants-and-guests.md [hunk 1]
```diff
@@ -66,6 +66,9 @@ This is a per-organizer policy. This setting controls whether people join a meet
...
 |**Invited users only**    |Only invited users and meeting organizers can join the meeting directly without waiting in the lobby. Everyone else, including authenticated users within the organization, guest users, users from trusted organizations, and anonymous users must wait in the lobby. On the Teams client meeting options page, it appears as "People I invite". Users added as a part of a distribution group will have to go through the lobby.      |
 
+ [!NOTE]
+> Trusted Organizations are domains that you allow federated communications with in Teams. If you enable **Allow all external domains** for external access in the Teams Admin Center, any authenticated user within any Teams organization will be trusted. If you choose to specify external domains that are allowed and block all others, the allowed domains become trusted organizations. Any blocked domain is considered to not be a trusted organization.
```
Comment (JohanFreelancer9) [inline Teams/meeting-policies-participants-and-guests.md:70]:
```suggestion
> Trusted organizations are domains that you allow federated communications with in Teams. If you enable **Allow all external domains** for external access in the Teams admin center, any authenticated user within any Teams organization will be trusted. If you choose to specify external domains that are allowed and block all others, the allowed domains become trusted organizations. Any blocked domain is considered to not be a trusted organization.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @cchiffersMSFT, kindly commit the minor suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/meeting-policies-participants-and-guests.md [hunk 2]
```diff
@@ -66,6 +66,9 @@ This is a per-organizer policy. This setting controls whether people join a meet
 |**Organizer only**    |Only meeting organizers can join the meeting directly without waiting in the lobby. Everyone else, including authenticated users within the organization, guest users, users from trusted organizations, and anonymous users must wait in the lobby. On the Teams client meeting options page, it appears as "Only me".          |
 |**Invited users only**    |Only invited users and meeting organizers can join the meeting directly without waiting in the lobby. Everyone else, including authenticated users within the organization, guest users, users from trusted organizations, and anonymous users must wait in the lobby. On the Teams client meeting options page, it appears as "People I invite". Users added as a part of a distribution group will have to go through the lobby.      |
 
+ [!NOTE]
```
Comment (JohanFreelancer9) [inline Teams/meeting-policies-participants-and-guests.md:69]:
```suggestion
 > [!NOTE]
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Apologies @cchiffersMSFT, one more, please.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@CarolynRowe PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**